extensions/amp-analytics/0.1/test-e2e/test-cbe.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {Key} from '../../../../build-system/tasks/e2e/e2e-types';
+
+describes.endtoend(
+  'custom browser events',
+  {
+    fixture: 'amp-analytics/custom-browser-events.html',
+    environments: ['single'],
+  },
+  (env) => {
+    let controller;
+
+    beforeEach(async () => {
+      controller = env.controller;
+    });
+
+    function sleep(ms) {
+      return new Promise((res) => setTimeout(res, ms));
+    }
+
+    it('click on input field and fropdown menu to focus and then blur', async () => {
+      debugger;
+      const inputField = await controller.findElement('#inputText');
+      const inputField2 = await controller.findElement('#inputText2');
+
+      await controller.click(inputField2);
+      await controller.click(inputField);
+
+      await sleep(1000);
+      await expect(
+        'https://foo.com/event?type=blur&eventId='
+      ).to.have.been.sent;
```

Comment:
I haven't seen this before. Can we assert the URL that a request was sent to in this way?

Reply:
Yes. This just checks that the appropriate request is sent. 
Other tests like `extensions/amp-video/0.1/test-eze/test-amp-video-analytics.js` also assert the request in a similar manner.

Reply:
Sounds good. Is it possible to include `eventId` it as part of the assertion?